uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/sick-request-ctrl.js

```diff
@@ -52,6 +53,7 @@ define([
         vm.loading.absenceTypes = true;
         initRequest();
 
+        vm.initParams = { absenceType: { is_sick: true } };
```

Comment:
its to filter in the AbsenceType.all call which is different for each leave type